File: support_tools/swigwin-3.0.12/Examples/test-suite/python/inherit_missing_runme.py
import inherit_missing

a = inherit_missing.new_Foo()
b = inherit_missing.Bar()
c = inherit_missing.Spam()

x = inherit_missing.do_blah(a)
if x != "Foo::blah":
    print "Whoa! Bad return", x

x = inherit_missing.do_blah(b)
if x != "Bar::blah":
    print "Whoa! Bad return", x

x = inherit_missing.do_blah(c)
if x != "Spam::blah":
    print "Whoa! Bad return", x

inherit_missing.delete_Foo(a)

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/inherit_missing_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'inherit_missing'

a = Inherit_missing.new_Foo()
b = Inherit_missing::Bar.new
c = Inherit_missing::Spam.new

x = Inherit_missing.do_blah(a)
if x != "Foo::blah"
  puts "Whoa! Bad return #{x}"
end

x = Inherit_missing.do_blah(b)
if x != "Bar::blah"
  puts "Whoa! Bad return #{x}"
end

x = Inherit_missing.do_blah(c)
if x != "Spam::blah"
  puts "Whoa! Bad return #{x}"
end
